app/models/damage.rb

```diff
@@ -0,0 +1,24 @@
+class Damage < ActiveRecord::Base
```

Comment:
In the app, we do have a "Reservation(we are going to change the name to Repair or something)" to create a 'damage reservation' in api for blocking the damage car from rental. The car will be released once the 'damage reservation' end. For the damage table, it is used to show that there is a damage event happened.